tests/functional/api/bulk/annotation_test.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+import pytest
+from h_matchers import Any
+
+
+@pytest.mark.usefixtures("with_clean_db")
+class TestBulkAnnotation:
+    def test_it_requires_authentication(self, make_request):
+        response = make_request(headers={"bad_auth": "BAD"}, expect_errors=True)
+
+        assert response.status_int == 404
```

Comment:
It'd have to be supported by the `make_request()` fixture but I think `app.post()` can do this in a one-er with something like:

```python
make_request(headers={...}, status=404)
```

Reply:
Yeah, we could but I like the separation of act / assert. The `expect_errors` is an annoying muddle of this already I think.